openedx_events/tests/test_producer_config.py

```diff
@@ -55,6 +55,26 @@ def test_enabled_disabled_events(self, mock_producer):
             call_args
         )
 
+    @patch("openedx_events.apps.logger")
+    @patch('openedx_events.apps.get_producer')
+    def test_events_with_custom_metadata_disabled(self, mock_producer, mock_logger):
```

Comment:
but you're using custom metadata, how is it disabled?

Reply:
Named updated to match what it does: signals are not sent by the handler on the consumer side if those are sent from the event bus.